README.md

```diff
@@ -1 +1,16 @@
-# spotavinyl
+## Spotavinyl App
+
+  This is the CodeFellows 301 final project of Laura Sammons, Maddie Mow, Joe Double.  It is a mashup between Spotify and eBay, helping users discover vinyl records based on their Spotify listening habits.
+
+## User Stories: MVP
```

Comment:
Great job on the user stories! I hope these were useful as you got started. At some point you'll see that these were a great starting point, and would have evolved into a much larger list as you began development. :-)